### ISSUE TITLE
Support parsing Oracle DROP TABLESPACE sql

### ISSUE THREAD

**Issue by FlyingZC** [apache/shardingsphere#27141]
# Background
Hi community,
This issue is for #26878.

ShardingSphere parser engine helps users parse a SQL to get the AST (Abstract Syntax Tree) and visit this tree to get SQLStatement (Java Object). Currently, we are planning to enhance the support for Oracle SQL parsing in ShardingSphere.

More details:
https://shardingsphere.apache.org/document/current/en/reference/sharding/parse/

# Task
This issue is to support more oracle sql parse, as follows:
```sql
DROP TABLESPACE undotbs_01;
```

```sql
EXPLAIN PLAN
  INTO my_plan_table
 FOR
SELECT last_name FROM employees;
```

```sql
EXPLAIN PLAN
  SET STATEMENT_ID = 'st1' FOR
SELECT last_name FROM employees;
```

```sql
EXPLAIN PLAN 
  SET statement_id = 'ex_plan1' FOR
SELECT phone_number FROM employees
 WHERE phone_number LIKE '650%';
```

```sql
EXPLAIN PLAN FOR
  SELECT last_name FROM employees;
```

# Process
1. First confirm that this is a correct oracle sql syntax, if not please ignore;
2. Compare SQL definitions in Oficial SQL Doc and ShardingSphere SQL Doc;
3. If there is any difference in ShardingSphere SQL Doc, please correct them by referring to the Official SQL Doc;
4. Run mvn install the current_file_module;
5. Check whether there are any exceptions. If indeed, please fix them. (Especially xxxVisitor.class);
6. Add new corresponding SQL case in SQL Cases and expected parsed result in Expected Statment XML;
7. Run SQLParserParameterizedTest to make sure no exceptions.

# Relevant Skills
1. Master JAVA language
2. Have a basic understanding of Antlr `g4` file
3. Be familiar with Oracle SQLs

**Comment by chakkk309** [comment]
Hi, I checked these SQL and found they are fixed by pr #27221 and #27940, so maybe we can mark this issue as completed.

**Comment by strongduanmu** [comment]
@chakkk309 Thank you for your test. Fixed by https://github.com/apache/shardingsphere/pull/27221 and https://github.com/apache/shardingsphere/pull/27940.
